src/decaylanguage/modeling/ampgentransform.py

```diff
@@ -42,7 +45,9 @@ def cplx_decay_line(self, lines):
 
     def decay(self, lines):
         (particle,) = lines[0].children
-        dic = {"name": str(particle), "daughters": []}
+        dic = OrderedDict()  # type: Dict[str, Any]
+        dic["name"] = str(particle)
+        dic["daughters"] = []
```

Comment:
This was the thing that broke when upgrading lark; it was reversing this item, and dicts are not reversible until about 3.7 or 3.8.

Reply:
Ah, but I did not touch that part of the code and I would bet that when I tried to upgrade Lark a while back (certainly over a year ago) the issues were with my decay file parser. But if things are solved for the whole package, then cool. It is not impossible that Lark itself had an issue at some point in history ... Excellent news 👍!